tests/PhpPresentation/Tests/Shape/AutoShapeTest.php

```diff
@@ -64,4 +69,83 @@ public function testType(): void
         self::assertInstanceOf(AutoShape::class, $object->setType(AutoShape::TYPE_HEXAGON));
         self::assertEquals(AutoShape::TYPE_HEXAGON, $object->getType());
     }
+
+    public function testPixelSetterComputesAdjAndAffectsHash(): void
```

Comment:
Move this test to https://github.com/PHPOffice/PHPPresentation/blob/master/tests%2FPhpPresentation%2FTests%2FWriter%2FPowerPoint2007%2FPptSlidesTest.php